system_info.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
Herre it would be nice to create a comment of wat the script does, whos the author 
Smth like this, https://bashcommands.com/bash-script-header there are many formats